glib/PtrArray.cs

```diff
@@ -91,6 +91,15 @@ public void Dispose ()
 		[DllImport ("libglib-2.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
 		static extern void g_free (IntPtr item);
 
+		static ListElementFree defaultFree;
+		protected static ListElementFree DefaultFree {
```

Comment:
Why not call it `g_free_func` or something instead of `DefaultFree` ? The `Default` in the name is a bit weird as it has no relation to the underlying function it's exposing